docs/data/system/getting-started/usage/Demo.tsx

```diff
@@ -10,11 +9,12 @@ export default function Demo() {
         display: 'flex',
         flexDirection: { xs: 'column', md: 'row' },
         alignItems: 'center',
-        bgcolor: 'background.paper',
-        overflow: 'hidden',
-        borderRadius: '12px',
-        boxShadow: 1,
-        fontWeight: 'bold',
+        bgcolor: 'background.default',
+        border: '1px solid',
+        borderColor: 'divider',
+        borderRadius: 2,
+        boxShadow: 2,
```

Comment:
This looked better before with:
```suggestion
        boxShadow: 1,
```

no? Or this could be even better:

```suggestion
        boxShadow: 0,
```

Reply:
I had entertained the option with no box-shadow at all but thought it'd be good to keep it there to demonstrate how folks could use the property. But yeah, without it's definitely better — these box-shadow styles are ugly 😬